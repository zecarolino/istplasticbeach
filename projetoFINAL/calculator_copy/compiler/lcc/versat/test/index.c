int mem[10], *base = mem;
int main() {
	base[6] = 9;
	return mem[6];
}
