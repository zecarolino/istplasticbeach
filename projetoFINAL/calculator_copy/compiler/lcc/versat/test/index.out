VCD info: dumpfile xtop.vcd opened for output.
9
Catched Trap at time  400 clock cycles:
